src/components/nav-bar/NavBar.tsx

```diff
@@ -3,9 +3,27 @@ import { signOut, useSession } from "next-auth/react";
 import Link from "next/link";
 import { useRouter } from "next/navigation";
 import { RxAvatar } from "react-icons/rx";
+import displayNotifications from "../notifcations/Notifications";
+import { useEffect, useState } from "react";
+import { fetchNotifications } from "../../app/services/userService";
 export default function NavBar() {
   const { data: session, status } = useSession();
   const router = useRouter();
+
+  const [notifications, setNotifications]= useState([]);
+
+  useEffect(() => {
+    if(session?.user) {
+
+      const data = fetchNotifications(session?.user._id);
+      data.then((res) => {
+        if (res.status === 200) {
+          setNotifications(res.data);
+        }
+      });      
```

Comment:
I don't think we need to check for the HTTP status of 200 here -> it's checked already in the service function. 
Why not use a try catch? 

Also why not use async/await?

Reply:
Oh I know why, because you're in useEffect and it complains that you can't use async. There are other examples in the code of where we use async in a useEffect - basically you defined the async function in the useEffect and then call the function (without the await).